src/Data/Audience.elm

```diff
@@ -34,6 +43,63 @@ type alias Audience =
     , folder : Maybe Int
     }
 
+audienceTypeToString : AudienceType -> String
+audienceTypeToString t =
+    case t of
+        Shared ->
+            "Shared"
+        Curated ->
+            "Curated"
+        Authored ->
+            "Authored"
+
+audienceTypeToIcon : AudienceType -> Html.Html msg
```

Comment:
This is view logic so I would consider to do not have it inside Data module.

Reply:
Yeah I thought it wasn't that big of a deal and my thought process was "ok so this is a function that works on `AudienceType`, so it makes sense to put that function there." (I thought this was a good idea after watching Evan's talk [the life of a file](https://www.youtube.com/watch?v=XpDsk374LDE)

Reply:
It doesn't hurt me much as well, just opinion, but I would just not expect function with output like `Html msg` in data module 🤷